sycl/include/sycl/detail/pi.h

```diff
@@ -347,6 +347,8 @@ typedef enum {
   PI_EXT_CODEPLAY_DEVICE_INFO_SUPPORTS_FUSION = 0x20005,
   PI_EXT_DEVICE_INFO_ATOMIC_FENCE_ORDER_CAPABILITIES = 0x20006,
   PI_EXT_DEVICE_INFO_ATOMIC_FENCE_SCOPE_CAPABILITIES = 0x20007,
+  // The number of max registers per block (device specific)
+  PI_EXT_CODEPLAY_DEVICE_INFO_MAX_REGISTERS_PER_WORK_GROUP = 0x20008
```

Comment:
This is an addition to the PI interface and as such you need to increment `_PI_H_VERSION_MINOR` and add a version note above it.

Reply:
Added. :+1: